docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -504,7 +504,8 @@ set()
   dynamicMetadata:set(filterName, key, value)
 
 Sets key-value pair of a *filterName*'s metadata. *filterName* is a key specifying the target filter name,
-e.g. *envoy.lb*. The type of *key* and *value* is *string*.
+e.g. *envoy.lb*. The type of *key* is *string*. The type of *value* is any Lua type that can be mapped
+to a metadata value, including a Lua table with key/values.
```

Comment:
I think we need an extended example here. Could you provide one?

And also we need to list the "acceptable" type for this: `MetadataMapHelper::loadValue` in here.

Thanks!